spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/SchedulerOperations.java

```diff
@@ -46,6 +46,29 @@
 	 * {@link org.springframework.cloud.dataflow.core.TaskDefinition} to be scheduled.
 	 * @param taskProperties  properties required for scheduling or launching a task.
 	 * @param commandLineArgs the command line args to be used when launching the task.
+	 * @param platform the name of the platform where the schedule should be created.
+	 * If left null the default platform will be used.
```

Comment:
We can add null or empty, instead of just `null`

Reply:
done